### PR TITLE
Update & Fix Swedish translation

### PR DESCRIPTION
**Pull request by el97**
The proper name for the file should be sv according to ISO 639. https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

### CODE REVIEW

**Review by MrBartusek — CHANGES_REQUESTED** [review]
Update file import

**Comment by el97** [comment]
That's it right?

File: src/localize/languages/sv.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  },
+  "editor": {
+    "entity": "Enhet",
+    "integration": "Integration",
+    "required": "Krävs",
+    "recommended": "Rekommenderas",
+    "override_headline": "Överrida titel",
+    "hide_when_no_warning": "Göm när det inte finns några varningar",
+    "hide_caption": "Dölj bildtext",
+    "disable_swiper": null,
```
Comment (MrBartusek) [inline src/localize/languages/sv.json:29]: What's up with that?

Comment (el97) [reply src/localize/languages/sv.json:29]: Yeah I wasn't sure how to translate disable_swiper so i left it untranslated.

**Review by MrBartusek — APPROVED** [review]
Thanks!

**Pull request merged**